src/pages/en/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,419 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Upgrade Guide
+description: How to upgrade your project to the latest version of Astro.
+i18nReady: false
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+
+This guide will help you migrate from Astro v1 to Astro v2.
+
+Need to upgrade an older project to v1? See our [older migration guide](https://github.com/withastro/docs/blob/aa7e4f568f418657eba297778729a8aee2a36409/src/pages/en/migrate.mdx).
+
+## Upgrade Astro
+
+1. You can check your version of Astro, and any official Astro integrations (`@astrojs/*`), in `package.json` at the root of your project. These may be in `devDependencies` and/or `dependencies`:
+
+    ```json "@astrojs/" {7}
+    //package.json
+    "devDependencies": {
+      "@astrojs/react": "^1.2.2",
+      "@astrojs/svelte": "^1.0.2",
+      "@astropub/icons": "^0.1.1",
+      "accessible-astro-components": "^1.4.5",
+      "astro": "1.9.2",
+      "react": "^18.2.0",
+      "react-dom": "^18.0.0",
+      "svelte": "^3.47.0"
+    },
+    "dependencies": {
+      "@astrojs/image": "0.13.0-beta.0",
+      "@astrojs/rss": "^2.0.0",
+      "@astrojs/vue": "^1.2.1",
+      "vue": "^3.2.41"
+    }
+    ```
+
+2. Update your project's version of Astro to the latest version using your package manager. If you're using Astro integrations, please also update those to the latest version.
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      # upgrades to Astro v2.x:
+      npm install astro@latest
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      # upgrades to Astro v2.x:
+      pnpm install astro@latest
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      # upgrades to Astro v2.x:
+      yarn add astro@latest
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+3. Check that you are running **at least Node `16.12.0`**. In v2.0, Astro dropped support for Node `14`.
+
+    ```sh
+    node -v
+    ```
+    
+    If necessary, [update your version of Node](#removed-support-for-node-14) in your local and deployment environments.
+
+## Update your code
```

Comment:
this is also confusing: how are these different from the next section of "Breaking Changes"? Are these different / duplicate, or do I need to read both?

Many of these seem like duplicates of the items below. If that's the case, then this section is safe to remove and then rewrite the next section to add:

```diff

## Astro 2.0 Breaking Changes

+ Astro v2.0 includes some breaking changes, as well as the removal of some previously deprecated features. If your project doesn't work as expected after upgrading to v2.0, check this guide for an overview of all breaking changes and instructions on how to update your codebase.

+ See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) for full release notes.

### Updated: Vite 4
```


Remember: A migration guide's goal isn't to summarize/highlights every single change that we've made. It's goal is to describe what a user needs to do to migrate their project from v1 -> v2, as concisely as possible. I get a vibe of "selling v2" from this section, but that's not necessary and gets in the way by making the upgrade seem more breaking/confusing than it needs.